mslib/mscolab/server.py

```diff
@@ -59,6 +59,8 @@
 migrate = Migrate(APP, db, render_as_batch=True)
 auth = HTTPBasicAuth()
 
+ARCHIVE_THRESHOLD = 30
```

Comment:
we should maybe make this configurable.

Reply:
Maybe. 30 days seems a tad short for me as a typical campaign may run longer.
New issue?